client/js/README.md

```diff
@@ -10,6 +11,12 @@ The Gradio JavaScript Client is available on npm as `@gradio/client`. You can in
 npm i @gradio/client
 ```
 
+Or, you can include it directly in your HTML via the jsDelivr CDN:
+
+```shell
+<script src="https://cdn.jsdelivr.net/npm/@gradio/client@0.19.4/dist/index.min.js"></script>
```

Comment:
I think we install the latest version here too instead of hardcoding a specific version, as below?